guides/upgrading/v2.0.md

```diff
@@ -172,8 +172,8 @@ And update the `attach` calls:
 Finally, replace references to `meta.stack` with `meta.stacktrace`:
 
 ```diff
-- Sentry.capture_exception(meta.error, stacktrace: meta.stack, extra: extra)
-+ Sentry.capture_exception(meta.error, stacktrace: meta.stacktrace, extra: extra)
+- Sentry.capture_exception(meta.reason, stacktrace: meta.stack, extra: extra)
++ Sentry.capture_exception(meta.reason, stacktrace: meta.stacktrace, extra: extra)
```

Comment:
This upgrade guide was for 2.0, so `meta.error` is still correct here.

Reply:
Ah okay, I reverted that change.